src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDeconstructTests.cs

```diff
@@ -505,7 +505,6 @@ static void Main()
     {
         int x, y;
         var type = ((x, y) = new C()).GetType();
```

Comment:
No, last I heard from LDM, deconstruction-assignment is an expression (with type void), while deconstruction-declaration will be a statement. 
See the notes from Mads (in email currently) from 5/20:
```

Deconstructing assignment
=========================

void expression, not a value.
```